frontend/src/services/user.service.ts

```diff
@@ -0,0 +1,18 @@
+import {User} from "../interfaces/User"
+import { CredentialResponse } from '@react-oauth/google';
+import axios from 'axios';
+
+export const googleLogin = async (credentialResponse: CredentialResponse) => {
+    return new Promise<User>(async (resolve, reject)  => {
+      console.log('credentialResponse', credentialResponse);
+      try{
+        const response = await axios.post<User>('/auth/google', credentialResponse.credential);
```

Comment:
put this url into a const in `utils/constants`

Reply:
done. added - AUTH_GOOGLE_URL , AUTH_GOOGLE_LOGIN_URL